File: modul 6/c/PRAK603-2410817110008-MuhammadAzmaAlFaqih.c
#include <stdio.h>

int main(){
  int a, b;
  scanf("%d %d", &a, &b);
  int matriks[2][a];
  if ( a != b ){
    printf("Jumlah tidak sama\n");
    return 0;
  }
  for (int i = 0; i < a; i++){
    scanf("%d", &matriks[0][i]);
  }
  for (int i = 0; i < a; i++){
    scanf("%d", &matriks[1][i]);
  }
  printf("\n");
  for (int i = 0; i < a; i++){
    printf("%d ", matriks[0][i]*matriks[1][i]);
  }
  printf("\n");
}

File: modul 6/python/PRAK603-2410817110008-MuhammadAzmaAlFaqih.py
n1, n2 = map(int, input().split())
if n1!=n2:
    print("Jumlah tidak sama")
    exit()
baris1 = []
baris2 = []
while(len(baris1) < n1):
    inputs = input()
    inputs = inputs.split()
    for i in inputs:
        baris1.append(int(i))
while(len(baris2) < n2):
    inputs = input()
    inputs = inputs.split()
    for i in inputs:
        baris2.append(int(i))
for i in range(n1):
    print(baris1[i]*baris2[i], end=' ')
print()